scripts/variantstore/wdl/RetryWithMoreMemory.wdl

```diff
@@ -0,0 +1,22 @@
+version 1.0
+
+workflow RetryWithMoreMemoryWorkflow {
```

Comment:
proof of concept WDL for "Retry with more memory", "successful" (i.e. failing in the right way with subsequent attempts being allocated more memory) run [here](https://job-manager.dsde-prod.broadinstitute.org/jobs/7ade1b8a-edb7-4011-a679-d92ec7e05aad).